rdmo/management/assets/js/components/import/ImportElement.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {isEmpty} from 'lodash'
+
+import {
+  WarningLink,
+  ErrorLink,
+  ShowLink,
+  AvailableLink,
+  LockedLink, AddLink, EditLink
+} from '../common/Links'
+import SelectCheckbox from './common/SelectCheckbox'
+import Errors from './common/Errors'
+
+import Warnings from './common/Warnings'
+import Fields from './common/Fields'
+import Form from './common/Form'
+
+
+const ImportElement = ({ config, element, importActions }) => {
+  const updateShowField = () => importActions.updateElement(element, {show: !element.show})
+  const toggleImport = () => importActions.updateElement(element, {import: !element.import})
+  const updateElement = (key, value) => importActions.updateElement(element, {[key]: value})
+  const toggleAvailable = () => importActions.updateElement(element, {available: !element.available})
+
+  return (
+    <li className="list-group-item">
+      <div className="pull-right">
+        {
+          (isEmpty(element.errors) && ('available' in element)) &&
+           <AvailableLink available={element.available}
+                          locked={element.locked} onClick={toggleAvailable}
+                          title={element.available ? gettext('Make unavailable')
+                                         : gettext('Make available')}/>
+        }
+        {
+          !isEmpty(element.warnings) &&
+          <WarningLink onClick={updateShowField} />
+        }
+        {
+          !isEmpty(element.errors) &&
+          <ErrorLink onClick={updateShowField} />
+        }
+        {
+          (element.changed && element.updated) &&
```

Comment:
This should not be this kind of link, it is confusing to users. We need something else to indicate that the element is created or updated. Before we used "created" in green. We could use `<span class="label label-success">create</span>`. It should also be on the left hand side after the URI.

Reply:
Looks great, tiny change: can we use `label-info` for `(element.changed && element.updated)`?